deployment/v3/testrig/apitestrig/install.sh

```
@@ -88,29 +97,80 @@ function installing_apitestrig() {
-  --set apitestrig.configmaps.apitestrig.ENV_USER="$ENV_USER" \
-  --set apitestrig.configmaps.apitestrig.ENV_ENDPOINT="https://$API_INTERNAL_HOST" \
-  --set apitestrig.configmaps.apitestrig.ENV_TESTLEVEL="smokeAndRegression" \
-  --set apitestrig.configmaps.apitestrig.reportExpirationInDays="$reportExpirationInDays" \
-  --set apitestrig.configmaps.apitestrig.slack-webhook-url="$slackWebhookUrl" \
-  --set apitestrig.configmaps.apitestrig.eSignetDeployed="$eSignetDeployed" \
-  --set apitestrig.configmaps.apitestrig.NS="$NS" \
-  $ENABLE_INSECURE
-
-  echo Installed apitestrig.
-  return 0
+  read -p "Is values.yaml for apitestrig chart set correctly as part of pre-requisites? (Y/n) : " yn;
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)